charts/pega/README.md

```diff
@@ -106,6 +106,13 @@ To support this option,
 
 •  Pass secrets directly to your deployment using your organization's recommend practices.
 
+##### Things to note in case of providing keystore, certificates for Enabling encryption of traffic between Ingress/LoadBalancer and Pod
+1. Have the CA certificate and keystore as a base64 encrypted string inside the secret manager(AWS Secret Manager, Azure Key Vault etc). [Check this](https://github.com/smootherbug/pega-helm-charts/tree/tomcat-encrypt/charts/pega#enabling-encryption-of-traffic-between-ingressloadbalancer-and-pod)
+2. Have the keystore password as plaintext
```

Comment:
remember period at the end of the sentence>>
Have the keystore password as plaintext*.*

Reply:
done